src/app/mysql-query-details/mysql-query-details.component.ts

```diff
@@ -104,6 +104,12 @@ export class MySQLQueryDetailsComponent extends CoreComponent implements OnInit
     }
 
     const query = this.queryDetails.Example.Query;
+    const size = this.queryDetails.Example.Size;
+    if (size > 0 && size > query.length) {
+        this.classicExplainError = this.jsonExplainError = 'Cannot explain truncated query. This query was '+size+' bytes long and was truncated to maximum size of '+query.length+' bytes.';
+        this.isExplainLoading = false;
+        return
+    }
```

Comment:
Do we need same code for mongo?

Reply:
Not really, as I'm not truncating mongo examples, added anyway to be consistent.